packages/eslint-plugin/src/rules/no-unsafe-call.ts

```diff
@@ -20,12 +20,12 @@ export default createRule<[], MessageIds>({
   meta: {
     type: 'problem',
     docs: {
-      description: 'Disallow calling a value with type `any`',
+      description: 'Disallow calling a value with type an `any` value',
```

Comment:
[Docs] Still this typo...

```suggestion
      description: 'Disallow calling a value with type `any`',
```